tamr_toolbox/notifications/emails.py

```diff
@@ -0,0 +1,206 @@
+"""Tasks related to creation of Email notifications"""
+import logging
+import smtplib
+import ssl
+
+from typing import Union, List, Optional
+from email.mime.text import MIMEText
+from smtplib import SMTPException
+from tamr_toolbox.notifications.common import monitor_job as monitor_job_common
+
+from tamr_unify_client import Client
+from tamr_unify_client.operation import Operation
+
+from tamr_toolbox.models.operation_state import OperationState
+from tamr_toolbox.utils.operation import get_details
+
+LOGGER = logging.getLogger(__name__)
+
+
+def _build_message(
+    *, message: str, subject_line: str, sender: str, recipients: List[str]
+) -> MIMEText:
+    """Builds email message in Multipurpose Internet Mail Extensions (MIME) format. MIME is an
+    Internet standard that extends the format of email messages
+
+    Args:
+        message: Body of email message
+        subject_line: subject of email
+        sender: email address of sender
+        recipients: list of emails to send message to
+
+    Returns:
+        Email as a string
+    """
+
+    # build email
+    msg = MIMEText(message)
+    msg["Subject"] = subject_line
+    msg["From"] = sender
+    msg["To"] = ", ".join(recipients)
+    return msg.as_string()
+
+
+def send_email(
+    *,
+    message: str,
+    subject_line: str,
+    sender_address: str,
+    sender_password: str,
+    recipient_addresses: List[str],
+    smtp_server: str,
+    smtp_port: str,
+    raise_error: bool = True,
+    use_tls: bool = False,
```

Comment:
Should `use_tls=True` be the default? Is there any additional configuration required to be able to use this?

Reply:
SSL and TLS are two different cryptographic protocols. You can use one or the other. I have no problem with TLS being the default if you want. TLS is more secure.